### PR TITLE
force: edges and labels are imported multiple times

### PR DESCRIPTION
**Pull request by uruuru**
During the component handling's dfs edges were potentially visited multiple times. This resulted in edges (and their labels) being added multiple times to the imported `FGraph`.

As a consequence, edge labels were positioned multiple times and ended up far away from their corresponding edges:

image

After this fix: 
image

Note that the positions are still not perfect but that's a different story.

### CODE REVIEW

File: test/org.eclipse.elk.alg.force.test/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: ELK Force Tests
+Bundle-SymbolicName: org.eclipse.elk.alg.force.test;singleton:=true
+Bundle-Version: 0.8.0.qualifier
+Automatic-Module-Name: org.eclipse.elk.alg.force.test
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Vendor: Eclipse Modeling Project
+Require-Bundle: com.google.guava;bundle-version="15.0.0",
```
Comment (soerendomroes) [inline test/org.eclipse.elk.alg.force.test/META-INF/MANIFEST.MF:9]: Why guava `15.0.0`?

Comment (uruuru) [reply test/org.eclipse.elk.alg.force.test/META-INF/MANIFEST.MF:9]: Simply copied a different plugin as template. 

We could make the version consistent across all plugins though :).

Comment (uruuru) [reply test/org.eclipse.elk.alg.force.test/META-INF/MANIFEST.MF:9]: @soerendomroes I ran a grep on the repo and it turns out that whenever the version is set, it is set to 15 (with the exception of two test plugins, which use 18). I'm still open for an update though.

Comment (le-cds) [reply test/org.eclipse.elk.alg.force.test/META-INF/MANIFEST.MF:9]: I'll create an issue for bumping the Guava version.

**Pull request merged**